src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -2441,13 +2441,14 @@ class TxQ_test : public beast::unit_test::suite
         fillQueue(env, alice);
 
         auto calcTotalFee = [&](
-            std::int64_t alreadyPaid, int numToClear = -1) -> std::uint64_t {
+            std::int64_t alreadyPaid, boost::optional<std::size_t> numToClear = boost::none)
```

Comment:
I don't usually notice missing includes unless the build breaks, since it's just as valid if they come in indirectly. But I see your point. Added.